### PR TITLE
small tweaks needed

### PR DESCRIPTION
**Pull request by gidden**
don't necessarily merge, just FYI to get tests passing

### CODE REVIEW

File: message_ix/reporting/pyam.py [hunk 1]
```diff
@@ -19,10 +20,11 @@ def as_pyam(scenario, year_time_dim, *quantities, drop=[], collapse=None):
     # Convert each of *quantities* individually
     dfs = []
     for qty in quantities:
-        df = qty.to_series() \
-                .rename('value') \
-                .reset_index() \
-                .rename(columns=IAMC_columns)
+        df = qty.to_series() if isinstance(qty, xr.DataArray) else qty
```
Comment (gidden) [inline message_ix/reporting/pyam.py:23]: this should be done using `Quantity` probably

Comment (khaeru) [reply message_ix/reporting/pyam.py:23]: Done as iiasa/ixmp@96d98cd.

File: tests/test_reporting.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def test_reporter(test_mp):
 
     # …and expected values
     vom = rep.get(rep.full_key('ACT')) * rep.get(rep.full_key('var_cost'))
-    assert_xr_equal(vom, rep.get(vom_key))
+    assert_series_equal(vom, rep.get(vom_key))
```
Comment (khaeru) [inline tests/test_reporting.py:65]: I think we should add `ixmp.testing.assert_qty_equal(a, b)` that tests and maybe casts a and b to the current value of `ixmp.reporting.utils.Quantity`. That would also avoid this:
https://github.com/iiasa/ixmp/blob/d74c49326cc98021037af0122a484e4bbcd84114/tests/test_reporting.py#L444-L448

**Comment by khaeru** [comment]
Please proceed by modifying the branch in iiasa/message_ix#206.

**Pull request closed without merging**